docs/documenter.php

```diff
@@ -179,9 +177,9 @@ static function (string $symbol) use ($namespace): bool {
 /**
  * @return list<string>
  */
-function get_all_namespaces(): array
+function get_all_components(): array
 {
-    return [
+    $components = [
```

Comment:
Future improvement is to try to guess all namespaces based on the folder structure (whilst removing the internal once).

Reply:
The main issue is with things like `Psl\Str\Byte`, `Psl\Encoding\Hex` .. etc.